### PR TITLE
Bug 1522590 – Hero Component Refinements

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: content-src/components/DiscoveryStreamComponents/List/List.jsx [hunk 1]
```diff
@@ -79,6 +79,7 @@ export function _List(props) {
 }
 
 _List.defaultProps = {
+  recStartingPoint: 0,
```
Comment (k88hudson) [inline content-src/components/DiscoveryStreamComponents/List/List.jsx]: can you add a comment about what this is?

File: content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx [hunk 2]
```diff
@@ -53,6 +54,16 @@ export class Hero extends React.PureComponent {
         source={truncateText(rec.domain, 22)} />
     ));
 
+    let list = (
+      <List
```
Comment (k88hudson) [inline content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx]: Can you just confirm this won't be weird in terms of the telemetry? (i.e. will it think we have another list component or something?)

File: content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx [hunk 3]
```diff
@@ -157,6 +157,8 @@ export class _DiscoveryStreamBase extends React.PureComponent {
         return (
           <ImpressionStats rows={rows} dispatch={this.props.dispatch} source={component.type}>
             <Hero
+              subComponentType={embedWidth >= 9 ? `cards` : `list`}
```
Comment (k88hudson) [inline content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx]: personally I would pass in row width and add this logic to `Hero` if we intend to keep it consistent across usages

Comment (gvn) [reply content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx]: Yeah, I can see it both ways I guess. I'm trying to keep the components fairly dumb and light on logic.

Comment (gvn) [reply content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx]: This is also the only way I can foresee Hero being embedded.

**Comment by gvn** [comment]
All feedback should be addressed! 🎉

**Pull request merged**